Assets/Scenes/SampleScene.unity

```diff
@@ -412,6 +412,10 @@ PrefabInstance:
       propertyPath: m_Name
       value: Player
       objectReference: {fileID: 0}
+    - target: {fileID: 5215622920849717728, guid: c9cb149ef1a6c1d49b10c17aad617566, type: 3}
+      propertyPath: m_TagString
+      value: Player
```

Comment:
The Player tag change needs to be applied to the Player prefab. Currently, this tag change is only on the Player prefab instance in `SampleScene.unity`. See [this page](https://docs.unity3d.com/Manual/EditingPrefabViaInstance.html) for information about applying an override to a prefab.